sql/core/src/test/resources/sql-tests/inputs/array.sql

```diff
@@ -119,3 +119,21 @@ select get(array(1, 2, 3), 0);
 select get(array(1, 2, 3), 3);
 select get(array(1, 2, 3), null);
 select get(array(1, 2, 3), -1);
+
+-- function array_compact
+select
```

Comment:
I think we should change this case to one corresponding to

```
val invalid_datatype_df = Seq(1, 2, 3).toDF("a")
```

others is ok to me